source/PrintData.cpp

```diff
@@ -27,6 +27,87 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 using namespace std;
 
+namespace {
+	template <class Type>
+	string ObjectName(const Type &object);
+
+	template <>
+	string ObjectName<Ship>(const Ship &object) { return object.ModelName(); }
+
+	template <>
+	string ObjectName<Outfit>(const Outfit &object) { return object.Name(); }
+
+	template <class Type>
+	void PrintObjectSales(const Set<Type> &objects, const Set<Sale<Type>> &sales, const string &name = "name", const string &saleName = "sales")
```

Comment:
It seems like you're always specifying the default arguments? Why have them in that case?

Reply:
No reason, I suppose.